test/ReactiveSocketResumabilityTest.cpp

```diff
@@ -34,10 +35,15 @@ TEST(ReactiveSocketResumabilityTest, Disconnect) {
 
   auto stats = std::make_shared<MockStats>();
 
+  auto requestHandler = std::make_unique<StrictMock<MockRequestHandler>>();
+  EXPECT_CALL(*requestHandler, socketOnConnected()).Times(1);
+  EXPECT_CALL(*requestHandler, socketOnDisconnected(_)).Times(1);
+  EXPECT_CALL(*requestHandler, socketOnClosed(_)).Times(1);
+
   auto socket = ReactiveSocket::fromClientConnection(
       defaultExecutor(),
       std::move(socketConnection),
-      std::make_unique<DefaultRequestHandler>(),
```

Comment:
I changed this from the NullRequestHandler (alias: DefaultRequestHandler) to the mock in order to add the EXPECT_CALL.  Is this okay?  The tests still passed, and it doesn't look to me that the NullRequestHandler was doing a lot that the Mock wouldn't.  Are my assumptions correct?